api/jwt/extract_token.go

```diff
@@ -20,10 +22,20 @@ func ExtractToken(r *http.Request) string {
 }
 
 func ExtractAccessTokenMetadata(r *http.Request) (*store.AccessDetail, error) {
-	token, err := VerifyAccessToken(r)
+	tkn := ExtractToken(r)
+
+	// This part is used to verify the token
```

Comment:
this is also unnecessary because the code already has good readability. IIRC, previously, the parsing and validate signing is on other files. The separation was what made the code hard to read. But now they are all in this block, so it is already easier to read.